data/process_query_results.py

```diff
@@ -3,165 +3,72 @@
 ## to be that of the tool_name_version for better readibility
```

Comment:
Would be nice to update this to reflect current state and add usage description here, with an example invocation command.

Reply:
Good thinking - updated.